keras_cv/models/resnet_v2.py

```diff
@@ -0,0 +1,365 @@
+      Output tensor for the residual block.
+    """
+    if name is None:
+        name = f"v2_block_{backend.get_uid('v2_block')}"
+
+    def apply(x):
+        use_preactivation = layers.BatchNormalization(
+            axis=BN_AXIS,
+            epsilon=1.001e-5,
+            name=name + "_use_preactivation_bn")(x)
+
+        use_preactivation = layers.Activation(
+            "relu", name=name + "_use_preactivation_relu")(use_preactivation)
+
+        if conv_shortcut:
+            shortcut = layers.Conv2D(4 * filters,
+                                     1,
+                                     strides=stride,
+                                     name=name + "_0_conv")(use_preactivation)
+        else:
+            shortcut = layers.MaxPooling2D(
+                1, strides=stride)(x) if stride > 1 else x
+
+        x = layers.Conv2D(filters,
+                          1,
+                          strides=1,
+                          use_bias=False,
+                          name=name + "_1_conv")(use_preactivation)
+        x = layers.BatchNormalization(axis=BN_AXIS,
+                                      epsilon=1.001e-5,
+                                      name=name + "_1_bn")(x)
+        x = layers.Activation("relu", name=name + "_1_relu")(x)
+
+        x = layers.ZeroPadding2D(padding=((1, 1), (1, 1)),
+                                 name=name + "_2_pad")(x)
+        x = layers.Conv2D(
+            filters,
+            kernel_size,
+            strides=stride,
+            use_bias=False,
+            name=name + "_2_conv",
+        )(x)
+        x = layers.BatchNormalization(axis=BN_AXIS,
+                                      epsilon=1.001e-5,
+                                      name=name + "_2_bn")(x)
+        x = layers.Activation("relu", name=name + "_2_relu")(x)
+
+        x = layers.Conv2D(4 * filters, 1, name=name + "_3_conv")(x)
+        x = layers.Add(name=name + "_out")([shortcut, x])
+        return x
+
+    return apply
+
+
+def V2Stack(filters, blocks, stride=2, name=None):
+    """A set of stacked residual blocks.
+    Args:
+        filters: integer, filters of the bottleneck layer in a block.
+        blocks: integer, blocks in the stacked blocks.
+        stride: default 2, stride of the first layer in the first block.
+        name: string, stack label.
+    Returns:
+        Output tensor for the stacked blocks.
+    """
+    if name is None:
+        name = f"v2_stack_{backend.get_uid('v2_stack')}"
+
+    def apply(x):
+        x = V2Block(filters, conv_shortcut=True, name=name + "_block1")(x)
+        for i in range(2, blocks):
+            x = V2Block(filters, name=name + "_block" + str(i))(x)
+        x = V2Block(filters, stride=stride,
+                    name=name + "_block" + str(blocks))(x)
+        return x
+
+    return apply
+
+
+def ResNetV2(
+    stackwise_filters,
+    stackwise_blocks,
+    stackwise_strides,
+    include_rescaling,
+    name="ResNetV2",
+    include_top=True,
```

Comment:
For this internal method, and the V1 equivalent, let's default `include_top` to `None` for consistency with the external API.

While this doesn't change behavior, it makes the behavior a bit more explicit

Reply:
My apologies -- I misspoke here. I meant let's make this a required parameter with no default.